src/bot/index.ts

```diff
@@ -77,7 +77,7 @@ export async function bot({ currency, family, mutation }: Arg = {}) {
         if (mutation) {
           spec = {
             ...spec,
-            mutations: spec.mutation.filter((m) =>
+            mutations: spec.mutations.filter((m) =>
```

Comment:
Nice catch @emmanuelm41!
cc @gre

Reply:
I tried to use it and it was failing haha